.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -0,0 +1,69 @@
+name: Scanner release offline vulnerability bundle update
+on:
+  schedule:
+  - cron: "0 */3 * * *"
+  push:
+    branches:
+    - scanner/*
+    - yli3/*
+
+jobs:
+  read-release-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Generate matrix JSON
+      id: output-matrix
+      run: |
+        EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+        echo "matrix<<$EOF" >> "$GITHUB_OUTPUT"
+        sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
+        echo "$EOF" >> "$GITHUB_OUTPUT"
+
+  upload-release-vulnerabilities:
+    needs: read-release-versions
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false # If one of the versions fails, it should not stop others from succeeding.
+      max-parallel: 1 # Jobs are memory intensive, so only run one at a time.
+      matrix:
+        version: ${{ fromJson(needs.read-release-versions.outputs.matrix).versions }}
+    env:
+      ROX_PRODUCT_VERSION: ${{ matrix.version }}
+    steps:
+    - name: Authenticate with Google Cloud
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v2
+
+    - name: Generate offline vulnerability bundle
+      continue-on-error: true
+      run: |
+        set -eu
```

Comment:
I notice if the curl fails, the matrix step is still green: https://github.com/stackrox/stackrox/actions/runs/7648811771/job/20842211701
Is there any way we can fix that? @tommartensen

Reply:
`continue-on-error: true` is superfluous. It would only be required if you had more steps after "Generate offline vulnerability bundle" that you want to run in any case.

Information on whether `continue-on-error` sets step status to green or not is conflicting, but we can try it out by removing the line.

Reply:
The other matrix jobs will run, because you set `fail-fast: false`.